### DISCUSSION TITLE
guides/monitoring/gnp-monitoring

### DISCUSSION THREAD

**Discussion by giscus** [ismoilovdevml/devops-journey#32] [General]
# guides/monitoring/gnp-monitoring

Serverlarni monitoring qilish yoxud Grafana/Prometheus Stack

https://devops-journey.uz/guides/monitoring/gnp-monitoring

**Comment by mamadaliyevunical** [comment]
Post juda foydali, yozgan insonlar baraka topsin, katta rahmat.
